### ISSUE TITLE
bluetooth connexion

### ISSUE THREAD

**Issue by coutud** [evilsocket/pwnagotchi#1091]
hi i have probleme to connect my iphone with my pwnagotchi 
my iphone find it but the pwnagotchi stay at bt not found

thanks

**Comment by Brets0150** [comment]
Did you try manually pairing to your phone from the pi?
https://pwnagotchi.ai/configuration/#bluetooth

**Comment by coutud** [comment]
> Did you try manually pairing to your phone from the pi?
> 
> https://pwnagotchi.ai/configuration/#bluetooth

Yes i tried but my phone fond it but the pwnagotchi screen write nf

BEST regards

**Comment by storm546** [comment]
same issue here but with android trying to connect manually after pairing outputs an error:
`Failed to connect: org.bluez.Error.Failed`
Does anyone have a fix for this?

**Comment by chunkywill** [comment]
i had a similar thing when i tried connecting but i did remove <mac address> and then try manually pair it again and then it worked

**Comment by chunkywill** [comment]
> same issue here but with android trying to connect manually after pairing outputs an error: `Failed to connect: org.bluez.Error.Failed` Does anyone have a fix for this?

i had a similar problem but did the comman remove  _your phones mac adress_
and then manually re paired it and it worked fine

**Comment by waterfallm** [comment]
@chunkywill can you confirm your exact actions here as i've tried similar without much success

**Comment by Pejda** [comment]
> @chunkywill can you confirm your exact actions here as i've tried similar without much success

I had similar problem and somehow believe it was solved thanks to updates.
I did "sudo apt update -y" and "sudo apt upgrade -y" and then "sudo apt dist-upgrade -y" 
After reboot i just did "sudo bluetoothctl remove [MAC]" - replace [MAC] with mac address of already paired device - if you did the pairing before updates.
Before updating my device, it was impossible to fix this for me.

**Comment by waterfallm** [comment]
> I had similar problem and somehow believe it was solved thanks to updates.
> I did "sudo apt update -y" and "sudo apt upgrade -y" and then "sudo apt dist-upgrade -y"
> After reboot i just did "sudo bluetoothctl remove [MAC]" - replace [MAC] with mac address of already paired device - if you did the pairing before updates.

Same after update altough i'm running witn version 1.6.2 no updates applpied phone does get pinged and paired but pwnagotchi always shows "NF"

Do i need anything else in my bluetooth config? will wrong tethering IP cause this?
